proto/src/domain/notes.rs

```diff
@@ -25,7 +25,7 @@ impl From<(u64, NoteEnvelope)> for note::NoteCreated {
         Self {
             note_id: Some(note.note_id().into()),
             sender: Some(note.metadata().sender().into()),
-            tag: note.metadata().tag().into(),
+            tag: u32::from(note.metadata().tag()).into(),
```

Comment:
I think we should add the missing conversions to `miden-base`.

Reply:
@hackaugusto nice idea, done: https://github.com/0xPolygonMiden/miden-base/pull/559

Reply:
Eventually (in a different PR), tag here would be `u32` as well, right